homeassistant/components/wirelesstag.py

```diff
@@ -271,7 +271,7 @@ def update(self):
     def device_state_attributes(self):
         """Return the state attributes."""
         return {
-            ATTR_BATTERY_LEVEL: self._tag.battery_remaining,
+            ATTR_BATTERY_LEVEL: '{:.0%}'.format(self._tag.battery_remaining),
```

Comment:
This will misbehave if other devices have a value between 1 and 100 for battery_remaining. Do all devices report battery_remaining as a decimal between 0 and 1?

Reply:
When you say "other devices" do you mean outside of wireless tag devices?  I can't comment what every one does, however, from what I have in my installation, other Battery values are reported as a percentage from 0 - 100 (that's what I see for my zwave / iPhone devices).  So, from what I see this will make it consistent between devices.

Reply:
I mean other wireless tag devices.

Reply:
They all report as a decimal, this will fix all WirelessTag devices to be this way.

This is from their example code:

```javascript
batDisplayPc.text((tag.batteryRemaining * 100).toFixed(0));
```

http://wirelesstag.net/jshtmlview.aspx?html=index.html&js=styles/client.js

and you can see they all devices report as decimal and they are doing this conversion as well for their UI.